content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -58,9 +58,29 @@ arguments.
 6. Click **Generate Visualization**.
 7. View generated visualization by scrolling down.
 
+## Using TFX visualizations
+
+<video width="960" height="540" controls>
+  <source src="/docs/videos/tfdv_example_with_taxi_pipeline.webm" type="video/webm">
+</video>
+
+1. On Pipelines page, click **[Sample] Unified DSL - Taxi Tip Prediction Model Trainer** to open Pipeline Details page.
+2. On Pipeline Details page, click **Create run**.
+3. On Create run page,
+    * Use a run name and an experiment name of your choice.
+    * Note that in **pipeline-root** field, use a bucket you have permission to write to.
```

Comment:
Does this bucket have to be GCS or can it be with another provider, e.g. AWS? Assuming it can be with any provider, here's some suggested wording for this line:

```
    * In the **pipeline-root** field, specify a storage bucket that you have permission to write to. For example, enter the path to a Google Cloud Storage bucket or an Amazon S3 bucket.
```

If it has to be GCS, then the wording can be:

```
    * In the **pipeline-root** field, specify a Cloud Storage bucket that you have permission to write to.
```

Reply:
Done